.travis.yml

```diff
@@ -143,12 +181,12 @@ before_install:
  - export BUILD_FLAGS="$BUILD_FLAGS -DPYVER=$PYMVER -DSIRF_URL=$PWD -DSIRF_TAG=$TRAVIS_COMMIT"
  # get SuperBuild
  - cd ..
- - git clone https://github.com/CCPPETMR/SIRF-SuperBuild --recursive -b master
+ - git clone https://github.com/CCPPETMR/SIRF-SuperBuild --recursive -b OSX_abs_paths
```

Comment:
do we want this? we then will have to remember to remove it later.

Reply:
yes, have it in my local merge, will push when approved